bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -419,12 +419,20 @@ impl Connection {
         self.send(&[0; 0]).map_ok(|_| self)
     }
 
+    /// Gets the number of bytes that are currently available in the buffer to be read.
+    pub fn available(&self) -> usize {
```

Comment:
I'm not sure this name is super clear. Maybe `read_buffer_len`? Or `read_buffered_len`?

Reply:
"read_buffer_len" sounds like it would be the maximum read buffer size, not the size of the currently buffered data. And "read_buffered_len" seems too close to "read_buffer_len".

Maybe I should just leave it as "peek"? Or "recv_available" / "available_to_recv"?

Reply:
I like "peek_len"! It clarifies the confusion without deviating too far from the original API name.